### PR TITLE
Fix JWT multi-auth usage

### PR DESCRIPTION
**Pull request by buger**
it was trying to work with uninitialized session object, ignoring the fact that it can be done only if based identity provided by JWT

Fix https://github.com/TykTechnologies/tyk/issues/2580

### CODE REVIEW

File: gateway/multiauth_test.go [hunk 1]
```diff
@@ -255,3 +258,69 @@ func TestMultiSession_BA_Standard_FAILAuth(t *testing.T) {
...
+	})[0]
+
+	LoadAPI(spec)
+
+	jwtToken := CreateJWKToken(func(t *jwt.Token) {
+		t.Claims.(jwt.MapClaims)["user_id"] = "user"
+		t.Claims.(jwt.MapClaims)["exp"] = time.Now().Add(time.Hour * 72).Unix()
+	})
+
+	key := CreateSession()
```
Comment (letzya) [inline gateway/multiauth_test.go]: @buger `// Deprecated: Use Test.CreateSession instead.`

Comment (buger) [reply gateway/multiauth_test.go]: Missed this change ;)

But test works, so that should be fine.

**Pull request merged**